### PR TITLE
This is the greeting

### PR DESCRIPTION
**Pull request by AdamPalmer1988**
Added Hello World

### CODE REVIEW

File: PointOfSale/PointOfSale/Program.cs [hunk 1]
```diff
@@ -1,2 +1 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿Console.WriteLine("Hello World");
```
Comment (wuhtangclan) [inline PointOfSale/PointOfSale/Program.cs:1]: nice job dude

Comment (AdamPalmer1988) [inline PointOfSale/PointOfSale/Program.cs:1]: This code is derivative

**Comment by WisePixie** [comment]
good code

File: PointOfSale/PointOfSale/Program.cs (hunk 1 above)
Comment (wuhtangclan) [inline PointOfSale/PointOfSale/Program.cs:1]: i love you guys

Comment (WisePixie) [reply PointOfSale/PointOfSale/Program.cs:1]: good code

**Review by wuhtangclan — APPROVED** [review]

**Pull request merged**